rules.d/50-udev-default.rules.in

```diff
@@ -83,7 +83,7 @@ KERNEL=="fuse", MODE="0666", OPTIONS+="static_node=fuse"
 
 # The static_node is required on s390x and ppc (they are using MODULE_ALIAS)
 KERNEL=="kvm", GROUP="kvm", MODE="@DEV_KVM_MODE@", OPTIONS+="static_node=kvm"
-
+KERNEL=="sev", GROUP="sev", MODE="0664"
```

Comment:
please add brief comment here, what this is

Reply:
Done.